### PR TITLE
Landing page changes to look more like GOV UK style

### PR DESCRIPTION
**Pull request by hammerfunk**

### CODE REVIEW

File: app/assets/stylesheets/frontend/forms.scss [hunk 1]
```diff
@@ -806,7 +806,7 @@ input[type="file"] {
 }
 
 // TODO temp
-.steps {
+.question-block .steps {
```
Comment (matthewford) [inline app/assets/stylesheets/frontend/forms.scss]: Merge rule `.question-block .steps` with rule on line 799

File: app/controllers/content_only_controller.rb [hunk 2]
```diff
@@ -23,4 +33,7 @@ def get_current_form
     )
   end
 
+  def is_landing_page
```
Comment (matthewford) [inline app/controllers/content_only_controller.rb]: Rename `is_landing_page` to `landing_page?`.

File: app/assets/stylesheets/frontend/forms.scss [hunk 3]
```diff
@@ -796,17 +796,15 @@ input[type="file"] {
 
 // Numbered steps for form sections
 // Taken from https://github.com/alphagov/calculators/blob/9181624cc9e00a079bea2141a650321f7183b6d0/app/assets/stylesheets/calculators/child_benefit.scss
-.steps {
+.question-block .steps {
```
Comment (matthewford) [inline app/assets/stylesheets/frontend/forms.scss]: Rule set contains (16/10) properties

File: app/controllers/content_only_controller.rb [hunk 4]
```diff
@@ -23,4 +33,7 @@ def get_current_form
     )
   end
 
+  def landing_page?
+    @landing_page? = true
```
Comment (matthewford) [inline app/controllers/content_only_controller.rb]: unexpected token tEQL

Comment (matthewford) [inline app/controllers/content_only_controller.rb]: yeah @hammerfunk remove the ?, `@landing_page = true`

File: app/views/layouts/application.html.slim [hunk 5]
```diff
@@ -26,46 +26,64 @@
...
+  form#search.site-search action="/search" method="get" role="search"
+    .content
+      label for="site-search-text"
+        ' Search
+      input.js-search-focus#site-search-text type="search" name="q" title="Search"
+      input.submit type="submit" value="Search"
 
 - content_for :content do
-  - unless admin_in_read_only_mode?
+  - if defined? @landing_page?
```
Comment (matthewford) [inline app/views/layouts/application.html.slim]: then this becomes: ` if landing_page?`

File: app/assets/stylesheets/frontend/forms.scss (hunk 3 above)
Comment (matthewford) [inline app/assets/stylesheets/frontend/forms.scss]: Rule set contains (16/10) properties

File: app/controllers/content_only_controller.rb (hunk 4 above)
Comment (matthewford) [inline app/controllers/content_only_controller.rb]: Then add a view helper to check for the presence of this variable

File: app/assets/stylesheets/frontend/forms.scss (hunk 3 above)
Comment (hammerfunk) [inline app/assets/stylesheets/frontend/forms.scss]: Ignore this cos I've extended their code with my own and have commented so

File: app/controllers/content_only_controller.rb (hunk 4 above)
Comment (dreamfall) [inline app/controllers/content_only_controller.rb]: @matthewford @hammerfunk `@landing_page == true` :eyeglasses:

**Pull request merged**